libmcount/record.c

```diff
@@ -1065,6 +1065,18 @@ int record_trace_data(struct mcount_thread_data *mtdp,
 	return 0;
 }
 
+static void write_map(FILE *out, struct uftrace_mmap *map,
+		      unsigned char major, unsigned char minor,
+		      uint32_t ino, uint64_t off)
+{
+	/* write prev_map when it finds a new map */
+	fprintf(out, "%"PRIx64"-%"PRIx64" %.4s %08"
+			PRIx64" %02x:%02x %-26u %s\n",
```

Comment:
Please do not break the format string into two lines.  I prefer one line for string constants (unless it has newlines in it).